### ISSUE TITLE
Content work for screens of new Income Limits tool

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#52158]
## Issue Description

Public Websites opened an intake request for content and IA support on the new Income Limits tool, which is set to launch by the end of Q1. They will be doing usability testing soon and would like the content on the screens to be close to what will eventually be on prod. 

Our team needs to make sure the content meets plain language and VA.gov style standards as well as has accurate eligibility information. 

We'll use this ticket to work within the team and will post the final content in the intake request. 

Sketch file to work from: https://www.sketch.com/s/a8506c94-a5bd-4a69-91ff-fe06545f3126

## Tasks
- [x] Rework content on product screens (Aliyah)
- [x] Review (Danielle)
- [ ] Copyedit (Megan)
- [x] Post final content for PW team in ticket #51132 (Aliyah)

## Acceptance Criteria
- [x] PW has final content for their user research

**Comment by aliyahblackmore** [comment]
Hi @DanielleThierryUSDSVA ! My review and changes to these screens are ready for you in this SharePoint document. 
[Income limits review SharePoint](https://dvagov-my.sharepoint.com/:w:/r/personal/aliyah_blackmore_va_gov/Documents/Attachments/Income%20Limits_Screens_AB_1.18.23.docx?d=w223aaf9652d145358dabc5f86c4402e9&csf=1&web=1&e=RLf59y)

I'll also pass a Word document version to you via Slack in case you have any trouble getting in to this!

I made a few comments in the document and flagged some things that may need to be reviewed from an accessibility perspective.

**Comment by aliyahblackmore** [comment]
**Update here**: I'm planning to do a review tomorrow of some of the IA suggestions (as they span content). I'll add this as an agenda item for the content sync with Danielle on 1/25.

**Comment by aliyahblackmore** [comment]
Update: There were a few adjustments made yesterday to the income limits screens by the designer. I took time this afternoon to review and adjustment any of my comments and suggestions in the SharePoint document. 
This is with Danielle.

Also, Danielle and I confirmed that we'll only review the IA recs for the primary entry points for the tool at this time/for the scope of the intake request.  I reviewed the primary entry point suggestions and made some comments and notes in the [SharePoint doc](https://dvagov-my.sharepoint.com/:w:/r/personal/aliyah_blackmore_va_gov/Documents/Attachments/Income%20Limits_Screens_AB_1.18.23.docx?d=w223aaf9652d145358dabc5f86c4402e9&csf=1&web=1&e=IPMizW).

**Additional next steps:** Once Danielle confirms the H1 of the tool page, I'll need to let IA know so that they can update the URL.

Noting here that I'll plan to review the other IA/content page recs next week - those aren't needed for the scope of the intake request.
